src/view/shell/desktop/LeftNav.tsx

```diff
@@ -62,39 +64,119 @@ import {
-        did: currentAccount!.did,
-        handle: currentAccount!.handle,
-      })}
-      style={[styles.profileCard, !isDesktop && styles.profileCardTablet]}
-      title={_(msg`My Profile`)}
-      asAnchor>
-      <UserAvatar
-        avatar={profile.avatar}
-        size={size}
-        type={profile?.associated?.labeler ? 'labeler' : 'user'}
-      />
-    </Link>
-  ) : (
-    <View style={[styles.profileCard, !isDesktop && styles.profileCardTablet]}>
-      <LoadingPlaceholder
-        width={size}
-        height={size}
-        style={{borderRadius: size}}
+  const onAddAnotherAccount = () => {
+    setShowLoggedOut(true)
+    closeEverything()
+  }
+
+  const profile = profiles?.find(p => p.did === currentAccount!.did)
+  const otherAccounts = accounts
+    .filter(acc => acc.did !== currentAccount!.did)
+    .map(account => ({
+      account,
+      profile: profiles?.find(p => p.did === account.did),
+    }))
+
+  return (
+    <View style={[a.my_md, gtTablet && [a.align_start, a.pl_md]]}>
+      {!isLoading && profile ? (
+        <Menu.Root>
+          <Menu.Trigger label={_(msg`Switch accounts`)}>
+            {({props}) => (
+              <Button label={props.accessibilityLabel} {...props}>
+                <UserAvatar
+                  avatar={profile.avatar}
+                  size={size}
+                  type={profile?.associated?.labeler ? 'labeler' : 'user'}
+                />
+              </Button>
+            )}
+          </Menu.Trigger>
+          <Menu.Outer>
+            {otherAccounts && otherAccounts.length > 0 && (
+              <>
+                <Menu.Group>
+                  {otherAccounts?.map(other => (
+                    <Menu.Item
+                      disabled={!!pendingDid}
+                      style={[a.gap_sm, {minWidth: 150}]}
+                      key={other.account.did}
+                      label={_(
+                        msg`Switch to ${sanitizeHandle(
+                          other.profile?.handle ?? other.account.handle,
+                          '@',
+                        )}`,
+                      )}
+                      onPress={() =>
+                        onPressSwitchAccount(other.account, 'SwitchAccount')
+                      }>
+                      <UserAvatar avatar={other.profile?.avatar} size={20} />
+                      <Menu.ItemText>
+                        {sanitizeHandle(
+                          other.profile?.handle ?? other.account.handle,
+                          '@',
+                        )}
+                      </Menu.ItemText>
+                    </Menu.Item>
+                  ))}
+                </Menu.Group>
+                <Menu.Divider />
+              </>
+            )}
+            <Menu.Item
+              label={_(msg`Add another account`)}
+              onPress={onAddAnotherAccount}>
+              <Menu.ItemText>
+                <Trans>Add another account</Trans>
+              </Menu.ItemText>
```

Comment:
I think it might look nice to have icons on these last two? Like a + and a sign-out icon. Can use `ItemIcon position='left'`

Reply:
I think I prefer it without icons? it's becoming very cluttered

<table>
  <tbody>
    <tr>
      <td><img width="200" alt="Screenshot 2024-12-23 at 23 06 59" src="https://github.com/user-attachments/assets/4bd4cf36-f668-45f0-a6ed-94c970011d3f" /></td>
      <td><img width="200" alt="Screenshot 2024-12-23 at 23 07 17" src="https://github.com/user-attachments/assets/fc13fa76-5681-47b4-96fc-3240d045a25e" /></td>
    </tr>
  </tbody>
</table>

Reply:
I usually love icons, but in some cases, like this, I find them optional... so makes sense to not have them.

BTW Are there any general UX guidelines to help answering those questions?